modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -62,30 +62,36 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 		</liferay-portlet:resourceURL>
 
 		<%
+		DDMFormInstance formInstance = ddmFormDisplayContext.getFormInstance();
+
+		boolean expired = false;
+
+		if (ddmFormDisplayContext.isExpirationDateEnabled()) {
+			expired = DDMFormInstanceExpirationStatusUtil.isFormExpired(formInstance, timeZone);
+		}
+
+		boolean preview = ddmFormDisplayContext.isPreview();
+		boolean showSuccessPage = ddmFormDisplayContext.isShowSuccessPage();
+
 		String languageId = ddmFormDisplayContext.getDefaultLanguageId();
 
 		Locale displayLocale = LocaleUtil.fromLanguageId(languageId);
-
-		DDMFormInstance formInstance = ddmFormDisplayContext.getFormInstance();
```

Comment:
@rodrigopaulino why did you decide to change the order of `displayLocale` and `formInstance`?

Reply:
To follow order of usage.